.travis.yml

```diff
@@ -0,0 +1,48 @@
+language: python
+sudo: false
+
+python:
+  - "3.6"
+  - "3.7"
+  - "3.7-dev" # 3.7 development branch
+  - "3.8-dev" # 3.8 development branch
+  - "nightly"
+
+matrix:
+  allow_failures:
+    - python: "3.7-dev"
+    - python: "3.8-dev"
+    - python: "nightly"
+    
+
+services:
+  - postgresql
+
+env:
+  global:
+    - ANYBLOK_DATABASE_DRIVER=postgresql
+    - ANYBLOK_DATABASE_USER=postgres
+    - ANYBLOK_DATABASE_NAME=travis_ci_test
+  jobs:
+    - PROJECT_PATH=II_setup-project
+    - PROJECT_PATH=III-01_external-blok
+
+virtualenv:
+  system_site_packages: false
+
+before_install:
+  - cd $PROJECT_PATH
```

Comment:
According travis errors, I guess that between each jobs, context is different so this is not propagated

Reply:
in fact, not sure you take attention you create [git submodules](https://git-scm.com/book/en/v2/Git-Tools-Submodules)  instead copy all the files, so you have to init submodules: ``git submodule init``, ``git submodule update``. I would prefer to **not** use git submodules...